tests/Operation/MapReduceTest.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
This directive is necessary to disable type coercion in `testConstructorOutArgumentTypeCheck`; all of the invalid values provided there can be coerced to strings.

Reply:
This means that the validation is weaker than before, since the values were not cast. Do we consider that it's the user's responsibility to send the right type and use strict mode if they wish to avoid this kind of error? In any case, it won't work with an int casted to string.

Reply:
Before this change, users could pass an int which we'd have to guard against. With the `string` type enforced by PHP, there are two scenarios: if the calling file has `strict_types` disabled, an int would be coerced to string; with `strict_types` enabled a `TypeError` is thrown. The end result is the same: the method will not be called with a non-string parameter.